src/sql/azdata.proposed.d.ts

```diff
@@ -976,4 +976,16 @@ declare module 'azdata' {
 			expiresOn?: number
 		}
 	}
+
+	export interface ConnectionInfoSummary {
+		/**
+		 * Indicates whether the server version is supported by ADS. The default value is true. If the value is false, ADS will show a warning message.
+		 */
+		isSupportedVersion?: boolean;
```

Comment:
It seems redundant to have both of these. Why not either just have the message field (and no message means it's supported) or put them into a single object `unsupportedVersionInfo` or something.

Reply:
if i don't have additional message to provide right now, but i still want to tell ADS that it is unsupported.

Reply:
Would suggest having it just be an optional object of type 

`unsupportedServerVersion?: { message?: string }`

Kind of funky to pass an empty object like that - but I'm not a big fan of having a bunch of related properties at the top level like this (for example, the aria stuff we have I wish we had done like VS Code did and put all in a single "accessibilityInfo" object)

Anyways not a huge deal so if you're strongly against it I guess this is fine...

Reply:
for scenarios with a lot more information, i would also prefer to have a object, but for this one, I'd like to keep it as is.